invokeai/frontend/web/src/features/deleteImageModal/components/DeleteImageButton.tsx

```diff
@@ -13,13 +13,18 @@ export const DeleteImageButton = memo((props: DeleteImageButtonProps) => {
   const { onClick, isDisabled } = props;
   const { t } = useTranslation();
   const isConnected = useAppSelector((s) => s.system.isConnected);
+  const imageSelectionLength: number = (useAppSelector((s) => s.gallery.selection) || []).length;
+  const labelMessage: string =
+    imageSelectionLength > 1
+      ? t('gallery.deleteImage_plural', { count: imageSelectionLength })
+      : t('gallery.deleteImage');
```

Comment:
The ` (Del)` suffix sneaked away here

Reply:
#### Toolbar button

![Screenshot 2024-04-14 194732](https://github.com/invoke-ai/InvokeAI/assets/12788435/142e2380-7b48-4112-a2c9-c9a9acfeb517)

![Uploading Screenshot 2024-04-14 194606.png…]()

#### Gallery menu

![image](https://github.com/invoke-ai/InvokeAI/assets/12788435/1f91f0eb-36a8-4313-b907-67b1f48e221c)

![image](https://github.com/invoke-ai/InvokeAI/assets/12788435/6278066c-b857-45fc-b5d2-d233b52c774b)

#### Modal

![image](https://github.com/invoke-ai/InvokeAI/assets/12788435/e8bab23b-529d-4352-a175-b45ef81c1717)

![image](https://github.com/invoke-ai/InvokeAI/assets/12788435/fc688ebf-ea78-4164-ace0-ee42a91e885c)